packages/postgresql/fix-hardcoded-paths.patch

```diff
@@ -1,12 +1,11 @@
-diff -uNr postgresql-13.0/contrib/pg_trgm/trgm_regexp.c postgresql-13.0.mod/contrib/pg_trgm/trgm_regexp.c
---- postgresql-13.0/contrib/pg_trgm/trgm_regexp.c	2020-09-21 23:47:36.000000000 +0300
-+++ postgresql-13.0.mod/contrib/pg_trgm/trgm_regexp.c	2020-10-19 21:31:14.749878910 +0300
+--- ./contrib/pg_trgm/trgm_regexp.c	2021-05-11 02:11:42.000000000 +0530
++++ ./contrib/pg_trgm/trgm_regexp.c.mod	2021-06-17 12:12:01.025156011 +0530
 @@ -2173,7 +2173,7 @@
  
  	{
  		/* dot -Tpng -o /tmp/source.png < /tmp/source.gv */
 -		FILE	   *fp = fopen("/tmp/source.gv", "w");
-+		FILE	   *fp = fopen("@TERMUX_PREFIX@/tmp/source.gv", "w");
++		FILE	   *fp = fopen("@TERMUX_PREFIX@/source.gv", "w");
```

Comment:
Can you explain why /tmp path component has been removed there? (few more similar issues below)

Reply:
Nice catch.

Reply:
Done ✅